loopdb/sqlc/migrations/000011_static_address_loop_in.up.sql

```diff
@@ -0,0 +1,54 @@
+-- static_address_swaps...
+CREATE TABLE IF NOT EXISTS static_address_swaps (
+    -- swap_hash is the primary identifier of the swap
+    swap_hash BLOB PRIMARY KEY,
+
+    -- swap_invoice is the invoice that needs to be paid by the server to
+    -- complete the loop-in swap.
+    swap_invoice TEXT NOT NULL,
+
+    -- last_hop is an optional parameter that specifies the last hop to be
+    -- used for a loop in swap.
+    last_hop BLOB,
+
+    -- quoted_swap_fee is the swap fee in sats that the server returned in
+    -- the swap quote.
+    quoted_swap_fee BIGINT NOT NULL,
+
+    -- deposit_outpoints is a list of outpoints in the format txid:idx that are
+    -- used for this swap.
+    deposit_outpoints BLOB NOT NULL,
+
+    -- htlc_tx contains the htlc transaction without signatures.
+    htlc_tx BLOB,
+
+    -- htlc_tx_fee_rate is the fee rate in sat/kw that is used for the htlc
+    -- transaction.
+    htlc_tx_fee_rate BIGINT NOT NULL,
+
+    -- htlc_timeout_sweep_tx contains the finalized htlc timeout sweep
+    -- transaction.
+    htlc_timeout_sweep_tx BLOB,
+
+    -- htlc_timeout_sweep_address contains the address the htlc timeout sweep
+    -- transaction sends funds to.
+    htlc_timeout_sweep_address TEXT
+);
```

Comment:
Maybe add creation time to simplify accounting when we implement it.

Reply:
`initiation_time` is part of the swaps table, which is populated here when the loop-in is created in the db.
https://github.com/lightninglabs/loop/pull/786/files#diff-0c1c627afb3395fd16e63089cc7622eb3c8ca8288a1fa7de657cb77f516cd35dR131

There are 3 tables to store the static address loop-in info:
1. `swaps` - all legacy info that the saddr-loopin has in common with old loop-ins
2. `htlc_keys` to store the htlc keys used for the htlc_tx
3. `static_address_swaps` to store static address loop-in related info